src/Aquifer.API/Endpoints/Resources/Content/CreateTranslation/Endpoint.cs

```diff
@@ -62,20 +74,36 @@ public override async Task HandleAsync(Request request, CancellationToken ct)
             Version = 1
         };
 
+        if (isCommunityUser) {
+            newResourceContentVersion.AssignedUserId = user.Id;
+        }
+
         var newResourceContent = new ResourceContentEntity
         {
             LanguageId = language.Id,
             ResourceId = baseContent.ResourceId,
             MediaType = baseContent.MediaType,
-            Status = ResourceContentStatus.TranslationNotStarted,
+            Status = isCommunityUser 
+                ? ResourceContentStatus.TranslationInProgress 
+                : ResourceContentStatus.TranslationNotStarted,
             Trusted = true,
             Versions = [newResourceContentVersion]
         };
 
         await dbContext.ResourceContents.AddAsync(newResourceContent, ct);
+        await historyService.AddStatusHistoryAsync(
+            newResourceContentVersion, 
+            isCommunityUser 
+                ? ResourceContentStatus.TranslationInProgress 
+                : ResourceContentStatus.TranslationNotStarted,
```

Comment:
Nit: we could do `newResourceContent.Status` here since the same ternary has already done the work above